.github/workflows/tests.yaml

```diff
@@ -0,0 +1,69 @@
+on: [push, pull_request]
+
+name: check & tests
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
```

Comment:
Is there a reason for not using `actions/checkout@v3`?

Reply:
Nope, thanks for pointing that out. I updated to `v3`.